### PR TITLE
allow to configure the saleschannel in the product export command

### PR DESCRIPTION
**Pull request by breaker92**
Currently if the shop has more then one saleschannel, you can not choose the saleschannel for which you want to export.

### CODE REVIEW

**Comment by breaker92** [comment]
Please consider to increase the CouplingBetweenObjects in phpmd:
Otherwise this issue won't be solved!

**Comment by a-laurowski** [comment]
> Please consider to increase the CouplingBetweenObjects in phpmd:
> Otherwise this issue won't be solved!

**Comment by breaker92** [comment]
@a-laurowski done

**Review by a-laurowski — CHANGES_REQUESTED** [review]

**Comment by a-laurowski** [comment]
> @a-laurowski done

@breaker92  Have you push something?

File: src/Command/ProductExportCommand.php [hunk 1]
```diff
@@ -44,31 +50,44 @@ class ProductExportCommand extends Command implements ContainerAwareInterface
...
         $this->addOption(self::UPLOAD_FEED_OPTION, 'u', InputOption::VALUE_NONE, 'Should upload after exporting');
         $this->addOption(self::PUSH_IMPORT_OPTION, 'i', InputOption::VALUE_NONE, 'Should import after uploading');
+        $this->addOption(self::SALESCHANNEL_OPTION, 's', InputOption::VALUE_OPTIONAL, 'ID of the saleschannel');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $feedService = $this->feedFactory->create($this->channelService->getSalesChannelContext());
+        $salesChannel = null;
+        if ($input->hasOption(self::SALESCHANNEL_OPTION)) {
```
[the remark this answers is not in the document]
Comment (breaker92) [reply src/Command/ProductExportCommand.php]: Hi @a-laurowski I've changed it to argument

**Comment by a-laurowski** [comment]
Hi @breaker92  
Thanks for the update! Looks good, only the one thing from code sniffer to fix.
Could you squash the commits into a single one? I could do this via GitHub but i'm afraid that in that case i will stole an authorship of that change which i obviously don't want to.

**Comment by breaker92** [comment]
> Hi @breaker92
> Thanks for the update! Looks good, only the one thing from code sniffer to fix.
> Could you squash the commits into a single one? I could do this via GitHub but i'm afraid that in that case i will stole an authorship of that change which i obviously don't want to.

Hi @a-laurowski 
the code sniffer issue is solved and the commits are squashed.

**Review by a-laurowski — APPROVED** [review]

**Comment by a-laurowski** [comment]
> > Hi @breaker92
> > Thanks for the update! Looks good, only the one thing from code sniffer to fix.
> > Could you squash the commits into a single one? I could do this via GitHub but i'm afraid that in that case i will stole an authorship of that change which i obviously don't want to.
> 
> Hi @a-laurowski
> the code sniffer issue is solved and the commits are squashed.

@breaker92  thanks, i'm merging the PR

**Pull request merged**